Lib/fontTools/ttLib/scaleUpem.py

```diff
@@ -141,6 +142,51 @@ def visit(visitor, varData):
             item[i] = visitor.scale(v)
 
 
+# COLRv1
+
+
+@ScalerVisitor.register(otTables.BaseGlyphPaintRecord)
+def visit(visitor, record):
+    oldPaint = record.Paint
+
+    transform = otTables.Affine2x3()
+    transform.populateDefaults()
+    transform.xy = transform.yx = transform.dx = transform.dy = 0
+    transform.xx = transform.yy = visitor.scaleFactor
+
+    scale = otTables.Paint()
+    scale.Format = 12 # PaintTransform
```

Comment:
you could use a PaintScaleUniform
https://github.com/fonttools/fonttools/blob/d275207dbc75d0dbe691f6f1c9fe3c9ffe01da5f/Lib/fontTools/ttLib/tables/otData.py#L1850-L1854

Reply:
They use F2Dot14...

Reply:
I can conditionally use it. But the transform is fine because all of them share it via offset, so it's only 1 byte worse than a PaintScaleUniform.

Reply:
I first used it and I hit an out-of-range error at compile-time...

Reply:
Working on it.

Reply:
right, I had forgotten

Reply:
https://github.com/fonttools/fonttools/commit/0c6b67e8eca0dbe4baa1038082ca473802e5123b